### ISSUE TITLE
Problem with Powermail and Power_Cond

### ISSUE THREAD

**Issue by chequille** [einpraegsam/powermail#792]
Hi Everybody,
unfortunately, after migration to Typo3 Version 10.x.x powermail_cond is not working anymore.
Of course I updated to the latest powermail 8.4.1 and powermal_cond working 8.2.2 under 10.x.x.

The form is displayed, but all conditions are not executed.
What I found during my research is to add a routeEnhancer to the site conf.yaml but it does not help.

THere is a script named https://....../anmeldeformular/powermail_cond.json, but if I browse this link I get an error "Opps an error occured.........." and in the log files there is written {"exception":"ReflectionException: Class PowermailCond\\Controller\\ConditionController does not exist in .......

Does somebody have an idea what I can check or what can be wrong?

Best regards,
Chequille

**Comment by einpraegsam** [comment]
Sounds for me like a caching or autoloading issue

**Comment by chequille** [comment]
WIll check autoloading.
But on my live site with Typo3 8.x.x and the according powermail versions, there is as well such a script call and it produces the same error.
The only difference is that in my live site the conditions are working.

**Comment by chequille** [comment]
Ok, I found  a way to get it working again.
I downgraded powermail_cond to the first Typo3 10.x.x compatible version; 8.0.1
With this version conditions worked again.
Afterwards, I installed again 8.2.2 and it is still working.

So far good for me, as I can go on with my migration to Typo3 Version 10.

Any idea what happend when doing this, downgrading and upgrading the version? Does it make any sense?

Best regards,
Chequille
